src/middlewared/middlewared/plugins/vm.py

```diff
@@ -1269,19 +1278,19 @@ def start(self, id, options):
         Dict(
             'options',
             Bool('force', default=False),
-            Int('timeout', default=90),
         ),
     )
     @job(lock=lambda args: f'stop_vm_{args[0]}_{args[1].get("force") if len(args) == 2 else False}')
     def stop(self, job, id, options):
```

Comment:
Can we have a docstring for this method that documents how it behaves when guest is unresponsive?